iOS_SDK/OneSignalSDK/Source/OneSignal.h

```diff
@@ -335,6 +335,48 @@ typedef NS_ENUM(NSInteger, OSNotificationPermission) {
 
 @end
 
+@interface OSDevice : NSObject
```

Comment:
Uhm I don't know which style are we following for this point. Usually, the definition goes on the .h but since we have it all mixed on the OneSignal.h maybe it does make sense to put it on the .m? let me know what you think

Reply:
For now I actually think this fine for `OSDevice` since everything is public access 

However in other situations the classes might have some things public access and other things that are not. In this case, we define a `.h` file and then creater a separate interface within the `OneSignal.h` with the public access things. 

Best example is the `OSInAppMessageAction` class we have. It has a .h and a .m but it has another interface within the `OneSignal.h` as well.